Backtracking/TugOfWar.js

```diff
@@ -0,0 +1,50 @@
+// https://www.geeksforgeeks.org/tug-of-war/
+
+// Given an array of n integers, we need to divide them into two subsets such that 
+// the difference between their sums is minimized.
+
+/**
+ * @param {number[]} arr - The array of integers.
+ * @returns {Array.<number[]>} Two subsets with minimized sum difference.
+ */
+
+
+function tugOfWar(arr) {
+    const n = arr.length;
+    let totalSum = arr.reduce((a, b) => a + b, 0);
+    let minimumDiff = Number.MAX_VALUE;
+    let result = [];
```

Comment:
Why not start out with `result = [arr, []]` (put `arr` into one subset, other subset is empty), and set `minimumDiff` to `totalSum`?